liota/lib/utilities/utility.py

```diff
@@ -272,6 +273,16 @@ def read_liota_config(section, name):
         log.warn('liota.conf file missing')
     return value
 
+def read_user_config(config_file_path):
+
+    config = ConfigParser.RawConfigParser()
+    config.optionxform = str
+    config.read(config_file_path)
+
+    user_config = dict(config.items('DEFAULT'))
```

Comment:
We never made use of any sections in sampleProp.conf. IOTCC and GENERICMQTT
 are just comments, not configuration file sections. All the properties in sampleProp.conf belong only to DEFAULT section.